README.md

```diff
@@ -42,15 +42,15 @@ With this `Metadata`, `Nodule` can load configuration using loader functions:
         import { Nodule, loadFromEnvironment } from 'nodule-config';
 
         const nodule = new Nodule(metadata);
-        nodule.loader(loadFromEnvironment).then(bottle => myInitFunc(bottle));
+        nodule.from(loadFromEnvironment).load().then(bottle => myInitFunc(bottle));
```

Comment:
I changed the API slightly:
 - I need a "build" function on my builder to initiative the async part
 - I chose to use `from` to differentiate loader configuration from the act of loading